ProcessManager.php

```diff
@@ -723,7 +723,7 @@ protected function bootstrapFailed(Connection $conn)
 
             foreach ($this->slaves as &$slave) {
                 $slave['keepClosed'] = true;
-                if(true === isset($slave['connection']) && true === is_object($slave['connection'])) {
+                if(true == isset($slave['connection']) && true == is_object($slave['connection'])) {
```

Comment:
We meant actually to completely omit `true ===` :P

`if(isset($slave['connection']) && is_object($slave['connection'])) {` or `if(isset($slave['connection']) && null !== $slave['connection']) {`